Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -197,6 +197,8 @@ class SubmissionDetailsPresenter {
             controller.player = player
             controller.view.accessibilityIdentifier = "SubmissionDetails.mediaPlayer"
             return controller
+        case .some(.basic_lti_launch):
```

Comment:
do we want `, .some(.external_tool)` in here?

Reply:
External tools are handled on line 153.